packages/odata-v4/src/uri-conversion/uri-value-converter.ts

```diff
@@ -2,6 +2,6 @@ import { createUriConverter } from '@sap-cloud-sdk/odata-common/internal';
 import { defaultDeSerializers } from '../de-serializers';
 
 /**
- * internal
+ * @internal
```

Comment:
I guess this is not internal, as I see it's exported in the index.

Reply:
Thanks, I thought it was a typo.